packages/alert/src/lib/alert.tsx

```diff
@@ -0,0 +1,125 @@
+import MUIAlert, {
+  AlertProps as MUIAlertProps,
+  AlertColor,
+} from '@mui/material/Alert';
+import { styled } from '@mui/material/styles';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import CheckCircleIcon from '@mui/icons-material/CheckCircleOutline';
+import Box from '@mui/material/Box';
+import { Typography } from '@mui/material';
+
+/* eslint-disable-next-line */
+export interface AlertProps extends MUIAlertProps {
+  severity?: AlertColor;
+  onClose?: () => void;
+  title?: string | undefined;
+  action?: React.ReactNode;
+}
+
+const StyledAlert = styled(MUIAlert)(({ theme }) => ({
+  borderRadius: '2px',
+  marginTop: '36px',
+  marginBottom: '12px',
```

Comment:
don't make sense to add margin to the root component

Reply:
Done